src/components/FormElements/ClaimantsPicker.tsx

```diff
@@ -110,7 +110,7 @@ export const ClaimantsPicker = ({
 
       <div>
         <Button
-          variant="secondary"
+          variant="tertiary"
```

Comment:
I'll revert [mine](https://github.com/stellar/laboratory/pull/897#discussion_r1669010736) to tertiary